src/main/java/org/jabref/logic/layout/format/ReplaceUnicodeLigaturesFormatter.java

```diff
@@ -0,0 +1,53 @@
+package org.jabref.logic.layout.format;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+import org.jabref.logic.l10n.Localization;
+import org.jabref.logic.layout.LayoutFormatter;
+import org.jabref.logic.util.strings.UnicodeLigaturesMap;
+import org.jabref.model.cleanup.Formatter;
+
+public class ReplaceUnicodeLigaturesFormatter implements LayoutFormatter, Formatter {
+
+    private Map<Pattern, String> ligaturesMap;
+
+    public ReplaceUnicodeLigaturesFormatter() {
+        ligaturesMap = new HashMap<>();
+        UnicodeLigaturesMap stringMap = new UnicodeLigaturesMap();
+        for (String key : stringMap.keySet()) {
+            ligaturesMap.put(Pattern.compile(key), stringMap.get(key));
+        }
+    }
+
+    @Override
+    public String getName() {
+        return Localization.lang("Replace Unicode ligatures");
+    }
+
+    @Override
+    public String getKey() {
+        return "remove_unicode_ligatures";
+    }
+
+    @Override
+    public String format(String fieldText) {
+        String result = fieldText;
+
+        for (Pattern key : ligaturesMap.keySet()) {
```

Comment:
I like the following code a bit more:
````
ligaturesMap.foreach((pattern, replacement) -> {
     result = pattern.matcher(result).replaceAll(replacement);
});
````
but this is probably a matter of taste....

Reply:
That's not possible because of side effects. `result` is overwritten in every iteration. You shouldn't do this in a lambda. Hence, I'd like to keep it as it is.

Reply:
Annotate ADR? 😇 -> Y-Statement instead of huge MADR?